packages/astro/src/index.ts

```diff
@@ -36,15 +32,7 @@ const createPlugin = (options?: SpotlightAstroIntegrationOptions): AstroIntegrat
         if (command === 'dev') {
           logger.info('[@spotlightjs/astro] Setting up Spotlight');
 
-          // Importing this plugin dynamically because for some reason, the top level import
-          // caused a client error because the source-map library code was bundled into the client
-          const { sourceContextPlugin } = await import('./vite/source-context');
-
-          config.vite.plugins = [
-            errorPageInjectionPlugin({ importPath: thisFilePath }),
-            sourceContextPlugin(),
-            ...(config.vite.plugins || []),
-          ];
+          config.vite.plugins = [spotlight({ showTriggerButton: false }), ...(config.vite.plugins || [])];
```

Comment:
No objections here just wanted to confirm that the previous behaviour of `source-map` code ending up in client bundles is no longer an issue?

Reply:
Good question, and I don't know. I'll check.

Reply:
Ran `pnpm build` in `demos/astro-playground` and inspected all `.js` files under `dist/client`. I see references to source map URLs but no bundled in source map so I think we are clear.

Reply:
This is now verified.

Reply:
Nope, it is here and I finally understand what you mean. Will follow up.